src/app/sessions/containers/sessions-listings-search/sessions-listings-search.component.ts

```diff
@@ -154,7 +154,11 @@ export class SessionsListingsSearchComponent implements OnInit {
 
     openAssignDialog() {
         this.dialog.open(AssignHearingDialogComponent, {
-            data: {hearingId: this.selectedHearing.id, startTimeDisplayed: !(this.selectedSessions.length > 1)}
+            data: {
+                hearingId: this.selectedHearing.id,
+                startTimeDisplayed: !(this.selectedSessions.length > 1),
+                startTime: (this.selectedSessions.length >= 1) ? this.selectedSessions[0].start : moment()
```

Comment:
Is this startTime prepared correctly here? I don't think that we should send value for that if that's multisession. It's being set on backend in assign action. The if statement is probably wrong here.

Reply:
it could be indeed empty as it defaults in the constructor.
In regard to being set on the backend - if that's not how it should work, we had a bug here. Have a look at initiateForm() removed lines 60 & 61 from AssignHearingDialog